include/grpcpp/server.h

```diff
@@ -120,6 +120,10 @@ class Server : public ServerInterface, private GrpcLibraryCodegen {
   int AddListeningPort(const grpc::string& addr,
                        ServerCredentials* creds) override;
 
+  /// NOTE: This is *NOT* a public API. The server constructors are supposed to
```

Comment:
So these server constructors are supposed to be private, not protected?

Reply:
Yeah, they are supposed to be `private` (but were changed to `protected` in https://github.com/grpc/grpc/pull/14517). So I am leaving it `protected` for now. I added a warning that this is going to be changed to `private` soon.